tools/infrasys/src/shared.rs

```diff
@@ -32,23 +32,27 @@ pub async fn get_stack_outputs(
     cfn_client: &CloudFormationClient,
     stack_name: &str,
     region: &str,
-) -> Result<Vec<rusoto_cloudformation::Output>> {
+) -> Result<Vec<aws_sdk_cloudformation::model::Output>> {
```

Comment:
Nit: Same here re: imports

Reply:
What we probably want is something like

```use aws_sdk_cloudformation::output::DescribeStacksOutput;```

and then have

```Result<Vec<DescribeStacksOutput>>```

Reply:
imported outputs at the top. Outputs is actually the type we want to return here.

Reply:
Ah, I see it changes from `DescribeStacksOutput` when you get to the `.stacks`.

`Output` seems very non-descriptive though and could conflict with something else down the road. Would it be better to do `use aws_sdk_cloudformation::model::{Output as CfnOutput, Parameter}`?